schema-extended/arxiv-extended.json

```diff
@@ -0,0 +1,707 @@
+            "type": "string",
+            "minLength": 1,
+            "examples": [
+              "Jr.",
+              "Sr.",
+              "I",
+              "II",
+              "III"
+            ]
+          },
+          "affiliations": {
+            "title": "List of affiliations",
+            "description": "Institutional affiliations",
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/affiliation"
+            },
+            "minItems": 1
+          },
+          "orcid": {
+            "title": "ORCID iD",
+            "allOf": [
+              {
+                "type": "string",
+                "pattern": "^\\d{4}-\\d{4}-\\d{4}-\\d{3}[0-9X]$"
+              },
+              {
+                "type": "string",
+                "$comment": "TODO: consider adding 'orcid' format here; would need to implement validator using checksum method described in https://support.orcid.org/hc/en-us/articles/360006897674-Structure-of-the-ORCID-Identifier"
+              }
+            ]
+          },
+          "author_id": {
+            "description": "arXiv author identifier, if available.",
+            "type": "string",
+            "pattern": "^[a-z]+_[a-z]{1}_[0-9]+$",
+            "$comment": "make generic?"
+          },
+          "is_submitter": {
+            "description": "Indicates whether this person is the submitter of this submission",
+            "type": "boolean"
+          }
+        },
+        "required": [
+          "full_name",
+          "last_name"
+        ],
+        "$comment": "Removed requirement for first name. Some people do not have a last name--optional too?"
+      },
+      "arxiv_status": {
+        "title": "arXiv posting status",
+        "description": "The status of this document version on arXiv.",
+        "type": "string",
+        "enum": ["posted", "withdrawn"],
+        "minLength": 1
+      },
+      "external_publication_status": {
+        "title": "External publication status",
+        "description": "The publication status of this document version.",
+        "type": "string",
+        "enum": ["preprint", "submitted", "accepted", "published"],
+        "$comment": "Is this enough?"
+      },
+      "reference_string": {
+        "title": "Reference string",
+        "description": "An unstructured reference string.",
+        "type": "string",
+        "minLength": 1
+      },
+      "source_info": {
+        "title": "Source information",
+        "description": "Information about the source input.",
+        "type": "object",
+        "properties": {
+          "source_format": {
+            "title": "Source format",
+            "type": "string",
+            "enum": ["tex", "pdftex", "pdf", "ps", "html", "docx", "none"]
+          },
+          "legacy_source_flags": {
+            "title": "Legacy source flags",
+            "type": "string",
+            "enum": ["1", "1A", "1S", "A", "AD", "AS", "D", "S", "S1"]
```

Comment:
what do these mean? do they have an overlap with other fields we're adding, such that it would make sense to translate them instead of preserving them literally?